frontend/js/src/common/brainzplayer/YoutubePlayer.tsx

```diff
@@ -380,6 +382,19 @@ export default class YoutubePlayer extends React.Component<YoutubePlayerProps>
     const leftBound =
       document.body.clientWidth - draggableBoundPadding * 2 - 350;
 
+    // Handle close button click
+    const handleClose = () => {
+      if (this.youtubePlayer) {
+        this.youtubePlayer.stopVideo();
+        // Clear playlist
+        this.youtubePlayer.cueVideoById("");
+      }
```

Comment:
From reading the component, it looks like this code is already taken care of in `componentDidUpdate` when the value of the `show` prop changes to a falsy value:
https://github.com/metabrainz/listenbrainz-server/blob/9c314adb66903f858b7346339c0740a9474bdf8a/frontend/js/src/common/brainzplayer/YoutubePlayer.tsx#L126-L130

So no need to duplicate the code here, as calling `onVisibilityChange` will make the parent component change the `show` prop.